### PR TITLE
runfix: Correctly map asset uploading state (WEBAPP-5934)

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/script/cryptography/CryptographyMapper.js [hunk 1]
```diff
@@ -186,15 +186,17 @@ export default class CryptographyMapper {
...
         key: uploaded.assetId,
         otr_key: new Uint8Array(uploaded.otrKey),
         sha256: new Uint8Array(uploaded.sha256),
         status: z.assets.AssetTransferState.UPLOADED,
         token: uploaded.assetToken,
       });
-    } else {
+    }
+
+    if (asset.hasOwnProperty('notUploaded') && notUploaded !== null) {
```
Comment (lipis) [inline src/script/cryptography/CryptographyMapper.js]: else if seems appropriate here

Comment (atomrc) [reply src/script/cryptography/CryptographyMapper.js]: Agreed

Comment (ffflorian) [reply src/script/cryptography/CryptographyMapper.js]: Seems appropriate, but isn't :smiley: it was changed in the last refactor, now I am changing it back, see https://github.com/wireapp/wire-webapp/pull/5831/files#diff-d2363003337179b68a0f317d6da77c58L198

Comment (ffflorian) [reply src/script/cryptography/CryptographyMapper.js]: Ok, @atomrc conviced me.

**Pull request merged**